src/breakpoints/index.ts

```diff
@@ -25,24 +25,24 @@ export class Breakpoints extends Panel {
     this.addWidget(header);
     this.addWidget(this.body);
 
-    header.toolbar.addItem(
-      'deactivate',
-      new ToolbarButton({
-        iconClassName: 'jp-DebuggerDeactivateIcon',
-        tooltip: `${this.isAllActive ? 'Deactivate' : 'Activate'} Breakpoints`,
-        onClick: () => {
-          this.isAllActive = !this.isAllActive;
-
-          // TODO: this requires a set breakpoint(bp: Breakpoints.IBreakpoint[]) method in the model
-          /*Array.from(this.model.breakpoints.values()).map((breakpoints: Breakpoints.IBreakpoint[]) => {
-            breakpoints.map((breakpoint: Breakpoints.IBreakpoint) => {
-              breakpoint.active = this.isAllActive;
-              this.model.breakpoint = breakpoint;
-            });
-          });*/
-        }
-      })
-    );
+    // header.toolbar.addItem(
```

Comment:
Should we just remove the code instead of commenting it?

Reply:
This is tracked in the git history, so we could still find the previous code.

We can also keep a reference to the commit to make it easier to find it back later on.